src/TMedia.cpp

```diff
@@ -42,7 +42,7 @@ TMedia::TMedia(Host* pHost, const QString& profileName)
     connect(mpNetworkAccessManager, &QNetworkAccessManager::finished, this, &TMedia::writeFile);
 }
 
-TMedia::~TMedia() {}
+TMedia::~TMedia() = default;
```

Comment:
Do you need to even mention a `default` destructor? :confused:

Reply:
Doesn't seem to hurt to point out that it should be default, and clang-tidy reckons it's the way to go.